### ISSUE TITLE
Any way to disable logs from terminal?

### ISSUE THREAD

**Issue by devfelipereis** [mageddo/dns-proxy-server#60]
First, thanks for all!

I'm using this image in my docker-compose file and when not running in detach mode, I see a lot of logs in the screen.

Is there any way to disable logs?

**EDIT**

I'm running in detach mode and using **docker logs** command to inspect my containers. A good solution for now.

**Comment by mageddo** [comment]
Thank you for appreciate DPS, currently there is no way to disable logs, anyway I'm working to bring this feature soon at version `2.3.4`, Bellow the description:

* Redirect logs from console to file
* Disable logs entirely
* Set log level

All this options will be configurable by JSON configuration, terminal arguments and system environment variable.

**Comment by spravo** [comment]
hey @devfelipereis.

If you want simple to disable log you should add the **logging driver** with the mode **none**.

example *docker.compose.yml*
```yml 
  nginx:
    image: nginx
    environment:
      - HOSTNAMES=test.com
    ports:
      - 80:80
  proxy:
   image: defreitas/dns-proxy-server
   logging:
      driver: none
    ports:
      - 5380:5380
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - /etc/resolv.conf:/etc/resolv.conf
    hostname: dns.mageddo
```

**Comment by devfelipereis** [comment]
@spravo So simple omg... Thank you so much!

**Comment by mageddo** [comment]
Looks like the problem is solved for you anyway I pre releleased a [new version](https://github.com/mageddo/dns-proxy-server/releases/download/2.4.0/dns-proxy-server-2.4.0.tgz) you can check the [documentation here](https://github.com/mageddo/dns-proxy-server/blob/master/docs/Features.md). I might be  useful for future users.

**Comment by mageddo** [comment]
Thank you guys

**Comment by rrrealman** [comment]
Hi, great thank for your service. I found that your log level option docs is invalid in latest versions. Currently I use 2.4.1 version docker image from `https://hub.docker.com/r/defreitas/dns-proxy-server/tags/`.

**Comment by mageddo** [comment]
@rrrealman I'm sorry I broke it when developed #48 anyway I fixed at 2.5.2

**Comment by rrrealman** [comment]
Never mind. Thank you for patch.
